content/v2.markdown

```diff
@@ -157,6 +157,23 @@ For example:
 
     https://api.dnsimple.com/v2/_/domains?per_page=100&page=2
 
+## Sorting
+
+API v2 results are implictely sorted according to policies that vary from endpoint to endpoint.
+
+You can decide your own sorting policy for each single API call via the `sort` param.
+This param accepts a pair of informations: the name of a field and the order (`asc` for ascending and `desc` for descending).
```

Comment:
_a pair of informations_ sounds odd to me. I'd probably go for something like _a key-value pair_. It also sounds like you can only pass one, when you can pass several. I'd tweak it to something like this:

> This param accepts a set of key-value pairs: the name of a field and the order criteria (`asc` for ascending and `desc` for descending).